CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 
+- Heroku-24 stack image preview support along with multi-architecture (arm64/amd64) support (https://github.com/heroku/heroku-buildpack-ruby/pull/1439)
```

Comment:
I'd suggest dropping this "multi-architecture support" / "arm64 support" bit. We don't have any real way to run or integration test classic buildpacks on arm64 hardware. We also don't currently have a way to run the resulting slugs on arm64 either.

Reply:
I am trying on the phrase "preview support," though clearly the **preview** didn't weight that much on your interpretation of the message.

I want to memorialize when that logic went live so that X years from now we could find this moment in time based on the changelog. I'm worried if I don't add it now, I'll never add it in the future. 

Also, people use the classic buildpack off of Heroku. Mostly dokku is the largest consumer.

So I would like the word "arm" to show up in the changelog. Maybe there's a better wording. Here's a stab:

> - Initial Heroku-24 base image support. Includes initial functionality to detect and use multi-architecture (arm64/amd64) binaries, though this functionality is not tested or verified (https://github.com/heroku/heroku-buildpack-ruby/pull/1439)

I think this is fairly explicit about what this PR is and isn't.